client/src/components/Navbar.vue

```diff
@@ -1,29 +1,109 @@
 <template>
     <md-toolbar class="md-primary" md-elevation="1">
-        <md-avatar>
-          <img src="../assets/thm_logo.png">
-        </md-avatar>
-        <h3 class="md-title" style="flex: 1">The Halo Mod</h3>
-          <router-link to="/"><md-button class="md-primary">Home</md-button></router-link>
+        <router-link to="/">
+          <md-button class="title-button">
+            <div class="combined-button">
+              <md-avatar>
+                <img class="logo" src="../assets/thm_logo.png">
+              </md-avatar>
+              <h1 class="md-title">TheHaloMod</h1>
+            </div>
+          </md-button>
+        </router-link>
+
+        <div class="right-side">
+
+          <md-button
+            href="https://github.com/halomod/TheHaloMod-SPA/issues/new"
+            target="_blank"
+          >
+            <div class="combined-button">
+              <md-icon class="svg-icon" :md-src="require('../assets/github-icon.svg')" />
+              <span>Report Issue</span>
+            </div>
+          </md-button>
```

Comment:
It's not 100% clear that the github icon is associated with the Report Issue. Might need to make the icon smaller and make it part of the Report Issue label (to the right of it).

Reply:
And since this is open source we may also just want a github icon that just sends you to the repo so... maybe no icon here?

Reply:
Steven said that he loves the toggle. So unless he says to remove it, seems like the way to go is to do what he said on that one. The identity comment is a little more confusing. Could you elaborate on that a little further?

Potentially removing the icon could help yes. I like it like that and hovering over it indicates it is combined with the report issue label. See below:

![image](https://user-images.githubusercontent.com/17774559/113499894-904be980-94ce-11eb-8049-946b36195fd1.png)

 But with this one as well, Steven specifically requested it. Might be something to bring up at the meeting on Monday.

Reply:
Haha I can appreciate that the identity thing is a little ambiguous. Basicallly, my thinking is that this is a utility with a very narrow focus and a general goal of fast comprehension / navigation / usage. It holds your hand and does everything for you. I think adding bells and whistles like personalized theming distracts from that goal, especially when the toggle for that feature is right next to things that are actually really important.

The question then becomes can we tuck it away somewhere where it doesn't share real estate with those things? But then you're designing more UI elements just for a light/dark mode toggle that no prospective user would really miss and you're going even further down the rabbit hole in the process.

Sooooooo... ya. Obviously just my opinion and if Steven wants it Steven wants it but that's my hot take. Like I said, not expecting you to change it just wanted to discuss it.

Reply:
And yea, we can talk about the icon during the meeting. I think github icons generally take you to github repos and its a little antipattern to use it for something else but w/e.

I will upgrade my review to a comment without explicit approval lol